test/highlight/basics.scala

```diff
@@ -26,19 +26,19 @@ object Hello {
   trait Test {
 // ^ keyword 
 //       ^ type
-     def meth(i: Int)(implicit x: Boolean) = ???
+    def meth(i: Int)(implicit x: Boolean) = ???
```

Comment:
I've made definitions in this trait to be of the same indentation as
```scala
trait Foo {
    def meth = ???
  def anonFun = ???
}
```
is parsed with an ERROR and this causes highlighting test to flack

We might want to open an issue on this, but I'm reluctant to doing this as `scalac` raises a warning for this code
```
-- Warning: foo.scala:3:2 ------------------------------------------------------
3 |  def anonFun = ???
  |  ^
  |  Line is indented too far to the left, or a `}` is missing
```